internal/cli/opsmanager/admin/backup/blockstore/create.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package blockstore
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	"go.mongodb.org/ops-manager/opsmngr"
+)
+
+var createTemplate = "Blockstore configuration '{{.ID}}' created.\n"
+
+type CreateOpts struct {
+	cli.OutputOpts
+	assignment           bool
+	encryptedCredentials bool
+	ssl                  bool
+	id                   string
+	label                []string
+	writeConcern         string
+	uri                  string
+	loadFactor           int64
+	maxCapacityGB        int64
+	store                store.BlockstoresCreater
+}
+
+func (opts *CreateOpts) init() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *CreateOpts) Run() error {
+	r, err := opts.store.BlockstoreCreater(opts.newBackupStore())
+	if err != nil {
+		return err
+	}
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newBackupStore() *opsmngr.BackupStore {
+	backupStore := &opsmngr.BackupStore{
+		AdminBackupConfig: opsmngr.AdminBackupConfig{
+			ID:           opts.id,
+			URI:          opts.uri,
+			WriteConcern: opts.writeConcern,
+			Labels:       opts.label,
+		},
```

Comment:
[nit] feel free to do this in the future as we still have more create to go but let's maybe create a shared `* CreateOpts` with this logic that we can reuse with all this admin commands

Reply:
I am going to implement this in the update command